lib/full-site-editing/block-templates.php

```diff
@@ -115,25 +115,27 @@ function _gutenberg_get_template_files( $template_type ) {
  * @return string Updated wp_template content.
  */
 function _inject_theme_attribute_in_content( $template_content, $theme ) {
-	$new_content = '';
-	$blocks      = parse_blocks( $template_content );
-
-	$updated_blocks = array_map(
-		function( $block ) use ( $theme ) {
-			if ( 'core/template-part' === $block['blockName'] && ! isset( $block['attrs']['theme'] ) ) {
-				$block['attrs']['theme'] = $theme;
-			}
+	$has_updated_content = false;
+	$new_content         = '';
+	$template_blocks     = parse_blocks( $template_content );
+
+	foreach ( $template_blocks as $key => $block ) {
+		if ( 'core/template-part' === $block['blockName'] && ! isset( $block['attrs']['theme'] ) ) {
+			// phpcs:ignore WordPress.Arrays.ArrayKeySpacingRestrictions.SpacesAroundArrayKeys
+			$template_blocks[$key]['attrs']['theme'] = $theme;
```

Comment:
Unsure why, but the `['attrs']` accessor in `$template_blocks[$key]['attrs']['theme']` was throwing a seemingly unrelated phpcs error

Reply:
What was the error? wasn't it that we need spaces around $key ?

Reply:
🤦 I was interpreting the error incorrectly. We needed spaces around $key. Thanks!